kv/main.go

```diff
@@ -64,6 +64,10 @@ var duration = flag.Duration("duration", 0, "The duration to run. If 0, run fore
 var benchmarkName = flag.String("benchmark-name", "BenchmarkBlocks", "Test name to report "+
 	"for Go benchmark results.")
 
+// Mongo flags. See https://godoc.org/gopkg.in/mgo.v2#Session.SetSafe for details.
+var mongoWMode = flag.String("mongo-wmode", "", "WMode for mongo session (eg: majority)")
```

Comment:
How many `wMode` options are there? Can you list the valid options in the help test (for the Mongo-newbies)?